cilium-cli/Makefile

```diff
@@ -1,28 +1,26 @@
 # Copyright Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-GO := go
-GO_BUILD = CGO_ENABLED=0 $(GO) build
-GO_TAGS ?=
-TARGET=cilium
+include ../Makefile.defs
+
+TARGET := cilium
+
+all: $(TARGET)
+
+.PHONY: all $(TARGET) local-release install clean test bench clean-tags tags
+
+
 INSTALL = $(QUIET)install
-BINDIR ?= /usr/local/bin
+BINDIR = /usr/local/bin
 CLI_VERSION=$(shell git describe --tags --always)
-THIS_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 CLI_MAIN_DIR?=./cmd/cilium
-STRIP_DEBUG=-w -s
-ifdef DEBUG
-	STRIP_DEBUG=
-endif
-GO_BUILD_LDFLAGS ?= $(STRIP_DEBUG) \
-	-X 'github.com/cilium/cilium/cilium-cli/defaults.CLIVersion=$(CLI_VERSION)'
+GO_BUILD_LDFLAGS += -X "github.com/cilium/cilium/cilium-cli/defaults.CLIVersion=$(CLI_VERSION)"
 
 TEST_TIMEOUT ?= 5s
 
 $(TARGET):
-	$(GO_BUILD) $(if $(GO_TAGS),-tags $(GO_TAGS)) \
-		-ldflags "$(GO_BUILD_LDFLAGS)" \
-		-o $(TARGET) \
+	$(GO_BUILD) \
+		-o $@ \
```

Comment:
So IIUC here `DEBUG` and `GO_TAGS` have no effects after this patch, but it is possible to achieve the same after the patch with `NOSTRIP` and `GO_TAGS_FLAGS` respectively? If correct, please document this in the commit message.

Reply:
Exactly, I updated the commit message to communicate the `DEBUG` -> `NOSTRIP` change and updated the Makefile to keep support for `GO_TAGS` by merging it with the `GO_TAGS_FLAGS` variable